Scripts/Scratchpad/Extensions/Disabled/keyboard.lua

```diff
@@ -0,0 +1,57 @@
+-- A minimal keypad meant for coordinate input
+
+local keypadMatrix = {
+  { "7", "8", "9", "N", "S"},
+  { "4", "5", "6", "W", "E"},
+  { "1", "2", "3", "'", "\""},
+  { "⌫", "0", {["↩"] = "\n"}, ".", {["␣"] = " "}},
+}
+
+local keyBMatrix = {
+  {"Q", "W", "E", "R", "T", "Y", "U", "I", "O", "P"},
+  {"A", "S", "D", "F", "G", "H", "J", "K" ,"L", "⌫"},
+  {"Z", "X", "C", "V", "B", "N", "M", {["␣"] = " "}, ".", ","},
+} 
+
+
+--local width = 30
+--local height = 30
```

Comment:
Any particular reason to have them commented out?

Reply:
Not that I can remember of. I was probably just experimenting with keyboard keys.